net-misc/tigervnc/tigervnc-1.7.0.ebuild

```diff
@@ -0,0 +1,183 @@
+		drm? ( x11-libs/libdrm )
+	)
+	!net-misc/vnc
+	!net-misc/tightvnc
+	!net-misc/xf4vnc"
+DEPEND="${RDEPEND}
+	amd64? ( dev-lang/nasm )
+	x86? ( dev-lang/nasm )
+	>=x11-proto/inputproto-2.2.99.1
+	>=x11-proto/xextproto-7.2.99.901
+	>=x11-proto/xproto-7.0.26
+	java? ( >=virtual/jdk-1.5 )
+	server?	(
+		virtual/pkgconfig
+		media-fonts/font-util
+		x11-misc/util-macros
+		>=x11-proto/bigreqsproto-1.1.0
+		>=x11-proto/compositeproto-0.4
+		>=x11-proto/damageproto-1.1
+		>=x11-proto/fixesproto-5.0
+		>=x11-proto/fontsproto-2.1.3
+		>=x11-proto/glproto-1.4.17
+		>=x11-proto/randrproto-1.4.0
+		>=x11-proto/renderproto-0.11
+		>=x11-proto/resourceproto-1.2.0
+		>=x11-proto/scrnsaverproto-1.1
+		>=x11-proto/videoproto-2.2.2
+		>=x11-proto/xcmiscproto-1.2.0
+		>=x11-proto/xineramaproto-1.1.3
+		>=x11-libs/xtrans-1.3.3
+		>=x11-proto/dri2proto-2.8
+		opengl? ( >=media-libs/mesa-10.3.4-r1 )
+	)"
+
+CMAKE_IN_SOURCE_BUILD=1
+
+src_prepare() {
+	if use server ; then
+		cp -r "${WORKDIR}"/xorg-server-${XSERVER_VERSION}/. unix/xserver
+	fi
+
+	eapply "${WORKDIR}"/patches/010_libvnc-os.patch
+	eapply "${WORKDIR}"/patches/030_manpages.patch
+	eapply "${WORKDIR}"/patches/055_xstartup.patch
+
+	default
+
+	if use server ; then
+		cd unix/xserver || die
+		eapply ../xserver118.patch
+		eautoreconf
+	fi
+}
+
+src_configure() {
+	use arm || use hppa && append-flags "-fPIC"
+
+	local mycmakeargs=(
+		-DENABLE_GNUTLS=$(usex gnutls)
+		-DENABLE_PAM=$(usex pam)
+		-DBUILD_JAVA=$(usex java)
+	)
+
+	cmake-utils_src_configure
+
+	if use server; then
+		cd unix/xserver || die
+		econf \
+			$(use_enable opengl glx) \
+			$(use_enable drm libdrm) \
+			--disable-config-hal \
+			--disable-config-udev \
+			--disable-devel-docs \
+			--disable-dmx \
+			--disable-dri \
+			--disable-dri3 \
+			--disable-glamor \
+			--disable-kdrive \
+			--disable-libunwind \
+			--disable-linux-acpi \
+			--disable-record \
+			--disable-selective-werror \
+			--disable-silent-rules \
+			--disable-static \
+			--disable-tslib \
+			--disable-unit-tests \
+			--disable-xephyr \
+			--disable-xinerama \
+			--disable-xnest \
+			--disable-xorg \
+			--disable-xvfb \
+			--disable-xwin \
+			--disable-xwayland \
+			--enable-dri2 \
+			--with-pic \
+			--without-dtrace \
+			--disable-present \
+			--disable-unit-tests
+	fi
+}
+
+src_compile() {
+	cmake-utils_src_compile
+
+	if use server; then
+		# deps of the vnc module and the module itself
+		local d subdirs=(
+			fb xfixes Xext dbe $(usex opengl glx "") randr render damageext miext Xi xkb
+			composite dix mi os hw/vnc
+		)
+		for d in "${subdirs[@]}"; do
+			emake -C unix/xserver/"${d}"
+		done
+	fi
+}
+
+src_install() {
+	cmake-utils_src_install
+
+	newicon "${DISTDIR}"/tigervnc.png vncviewer.png
+	make_desktop_entry vncviewer vncviewer vncviewer Network
+
+	if use server ; then
+		emake -C unix/xserver/hw/vnc DESTDIR="${D}" install
+		if ! use xorgmodule; then
+			rm -r "${D}"/usr/$(get_libdir)/xorg || die
```

Comment:
double-slashing, plus prefer ED. Make this:
`rm -r "${ED%/}"/usr/$(get_libdir)/xorg || die`